xanitizer.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analyzers
 description=Import Xanitizer security reports
 homepageUrl=https://github.com/RIGS-IT/sonar-xanitizer
-archivedVersions=1.3.2,1.3.3,1.4.0,1.5.0
-publicVersions=2.0.0
+archivedVersions=1.3.2,1.3.3,1.4.0,1.5.0,2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=com.rigsit
 defaults.mavenArtifactId=sonar-xanitizer-plugin
 
+2.1.0.description=Support for SonarQube 7.9, Added support for JavaScript and TypeScript, Added several new Java problem types
```

Comment:
Release notes mention Xanitizer 5.0. I don't remember the details (& too lazy to look) so honest question here: did you upgrade an underlying analyzer here? I.e. should this description mention upgrading to X5.0? Or does the user run that independently & import the reports?

Reply:
> Release notes mention Xanitizer 5.0. I don't remember the details (& too lazy to look) so honest question here: did you upgrade an underlying analyzer here? I.e. should this description mention upgrading to X5.0? Or does the user run that independently & import the reports?

Dear Ann,

Xanitizer is an external analyzer (which the user runs independently from SQ) and the plugin parses its report and creates issues from it. So I think there is no need to reference this in the description, especially since it is not mandatory to update, because the plugin also supports older versions.

Best regards,
Norman